components/bl-lottieAnimation-component/src/index.js

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useRef } from 'react';
+import { destroy, loadAnimation, play, stop } from './lottie.min';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function lottieAnimation({ component, eventHandlers }) {
+  const { display, classList, data, isLoop, type, isStopped } = component;
+  const { onHover, onUnhover } = eventHandlers;
+
+  const container = useRef(null);
+
+  useEffect(() => {
+    loadAnimationHandler(container, type, isLoop, data);
+
+    if (isStopped) {
+      stop();
+    }
+  }, [data]);
+
+  component.playAnimation = () => {
+    play();
+  };
+
+  component.stopAnimation = () => {
```

Comment:
What if we add an action to change the current animation? I know we could do it through data binding but action also could be useful.